modules/ROOT/pages/update-modules.adoc

```diff
@@ -0,0 +1,23 @@
+= Update your Modules
+
+Because Anypoint Studio allows you to modularize your projects, you can use the *Manage dependencies* wizard to manage the dependencies of your Mule project.
+
+To update check for updates and update the version of the modules in your project:
+
+. In the top menu bar, select the *Manage dependencies* icon image:manage-dependencies.png[20,20]
+. Under *Mule Project*, select *Modules*.
++
+image::manage-modules.png[]
++
+When Studio loads this wizard for the first time, it automatically checks for new versions of your modules.
+. Select the module that you want to update and click *Update version*.
++
+image::update-module-version.png[]
++
+Additionally, you can select the *Update all dependencies to latest* button to update all dependencies to their latest versions.
++
+image::update-all-modules.png[]
+. Select *Apply and close*.
+
+Studio updates your selected dependencies to their latest version available.
```

Comment:
"Studio updates your selected dependencies": I would change "dependencies" to "modules", since that is the particular type of dependency that this topic is about.